rules/windows/execution_suspicious_powershell_imgload.toml

```
@@ -16,7 +16,79 @@ index = ["logs-endpoint.events.*", "winlogbeat-*", "logs-windows.*"]
 language = "eql"
 license = "Elastic License v2"
 name = "Suspicious PowerShell Engine ImageLoad"
-note = """## Config
+note = """## Triage and analysis
+
+### Investigating Suspicious PowerShell Engine ImageLoad
+
+PowerShell is one of the main tools system administrators use for automation, report routines, and other tasks. This
+makes it available for use in various environments, and creates an attractive way for attackers to execute code.
+
+Attackers can use PowerShell without having to execute `PowerShell.exe` directly. This technique is often called
+"PowerShell without PowerShell" and works by using the underlying System.Management.Automation namespace and can bypass
+application whitelisting and PowerShell security features.
```

This sentence is a bit of a run-on (several clauses all joined by just "and"). @nastasha-solomon's suggestion above would help, or here's another possibility. This also changes "whitelisting" to "allowlisting," as long as that doesn't change technical meaning here:
```suggestion
Attackers can use PowerShell without having to execute `PowerShell.exe` directly. This technique, often called
"PowerShell without PowerShell," works by using the underlying System.Management.Automation namespace and can bypass
application allowlisting and PowerShell security features.
```